src/common/procedure/src/local.rs

```diff
@@ -87,11 +94,14 @@ impl ProcedureMeta {
             state_sender,
             state_receiver,
             children: Mutex::new(Vec::new()),
+            start_time_ms: common_time::util::current_time_millis() as u64,
```

Comment:
Seems the creation of `ProcedureMeta` doesn't mean the starting of a procedure. So I think it's better to set the start time elsewhere.